client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -112,20 +119,18 @@ const GoalsStep: Step = ( { navigation, flow } ) => {
 		} );
 	};
 
-	const getStepSubmissionHandler =
-		( action: string, eventProps: Record< string, unknown > = {} ) =>
-		() => {
```

Comment:
This is a logic change. I removed `eventProps` because it's always empty. You can see both calls below in lines 124 and 125.